### PR TITLE
UNI-490: HTM Studio drop down menus 

### PR DESCRIPTION
**Pull request by marionleborgne**
Fixes: 
* UNI-490 HTM Studio drop down menus have meaningless/ useless items
* UNI-491 Link in about box should say "Terms and Conditions" not "License"
* UNI-489 About box has extraneous title after clicking license link
* UNI-498 add a "Report bug" submenu 

And should help for the windows build. @oxtopus let me know if this works for you.

### CODE REVIEW

**Comment by oxtopus** [comment]
👍

File: unicorn/app/main/MainMenu.js [hunk 1]
```diff
@@ -15,64 +15,176 @@
...
+        label: 'Provide Feedback',
+        click() {
+          let url = 'http://numenta.com/htm-studio#feedback';
+          electron.shell.openExternal(url);
+        }
+      },
+      {
+        label: 'Report Bug',
+        click() {
+          let url = 'http://github.com/numenta/numenta-apps/issues/new';
```
Comment (lscheinkman) [inline unicorn/app/main/MainMenu.js]: Are you sure you want to use `github` issues for this?
Only users with `github` credentials are allowed to create `github` issues.

File: unicorn/app/main/MainMenu.js [hunk 2]
```diff
@@ -15,64 +15,176 @@
...
-  // Learn More
-  helpMenu.submenu.splice(0, 1); // Remove old behavior
-  helpMenu.submenu.push({
-    label: 'Lean More',
-    click() {
-      let url = 'http://numenta.com/htm-studio';
-      electron.shell.openExternal(url);
-    }
-  });
+if (process.platform === 'darwin') {
```
Comment (lscheinkman) [inline unicorn/app/main/MainMenu.js]: No `About`/`Edit`/`Window` menus on windows?

File: unicorn/app/main/MainMenu.js (hunk 1 above)
Comment (marionleborgne) [inline unicorn/app/main/MainMenu.js]: i'll make this send an email to `support@numenta.com`

File: unicorn/app/main/MainMenu.js (hunk 2 above)
Comment (marionleborgne) [inline unicorn/app/main/MainMenu.js]: There is an about menu on windows, but with just one submenu: to see the version & Terms and Conditions. The rest of the about submenus like Services or Show All are OSX specific.

I can add a `if (process.platform === 'windows')` and add more entries for About / Edit / Window

Comment (lscheinkman) [inline unicorn/app/main/MainMenu.js]: At least for the `Edit` menu, for the same reasons we have `Edit` menu on Mac

File: unicorn/app/main/MainMenu.js [hunk 3]
```diff
@@ -77,7 +78,10 @@ let crossPlatformMenu = [
       {
         label: 'Report Bug',
         click() {
-          let url = 'http://github.com/numenta/numenta-apps/issues/new';
+          let url = `mailto:htm-studio@numenta.com?
+          subject=HTM Studio ${VERSION} bug
+          &body=DO NOT REMOVE THIS INFORMATION: HTM Studio version ${VERSION}.
+          Please describe the steps to reproduce the bug below: `;
```
Comment (lscheinkman) [inline unicorn/app/main/MainMenu.js]: body text appears as one single line with lots of spaces:
```
DO NOT REMOVE THIS INFORMATION: HTM Studio version 0.0.2.          Please describe the steps to reproduce the bug below:
```

File: unicorn/app/main/MainMenu.js [hunk 4]
```diff
@@ -77,7 +78,10 @@ let crossPlatformMenu = [
       {
         label: 'Report Bug',
         click() {
-          let url = 'http://github.com/numenta/numenta-apps/issues/new';
+          let url = `mailto:htm-studio@numenta.com?
+          subject=HTM Studio ${VERSION} bug
```
Comment (lscheinkman) [inline unicorn/app/main/MainMenu.js]: `subject` field was not passed to the email app, probably due to spaces in the URL

**Comment by marionleborgne** [comment]
@lscheinkman the issue was indeed due to blank spaces after formatting

**Comment by lscheinkman** [comment]
👍 LGTM

**Pull request merged**